<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="8" column="1">
    <widget class="WColorButton" name="pbBgColor">
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>When Store alpha channel option is unchecked, for files with alpha channel, defined transparent color will be used as background color</string>
     </property>
     <property name="text">
      <string>.</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QRadioButton" name="rbStoreAlpha">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save alpha channel (transparency)&lt;/p&gt;&lt;p&gt;If unchecked, for files with alpha channel, defined background color will be used&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-left: 30px</string>
     </property>
     <property name="text">
      <string>&amp;Use alpha channel</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QRadioButton" name="rbUseBgColor">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Don't save alpha channel (transparency)&lt;/p&gt;&lt;p&gt;If checked, for files with alpha channel, defined background color will be used&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-left: 30px</string>
     </property>
     <property name="text">
      <string>Use bac&amp;kground color</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QCheckBox" name="cbForceConvertsRGB">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Convert file to sRGB ICC profile&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-left: 30px</string>
     </property>
     <property name="text">
      <string>Convert to sRGB</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Advanced options</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="cbInterlacing">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Use interlacing when publishing on internet&lt;/p&gt;&lt;p&gt;For low bandwidth connection, this option allows browsers to load an early degraded version of the whole image as soon as possible and then progressively renders the image during loading&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-left: 30px</string>
     </property>
     <property name="text">
      <string>Interlacing</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="cbSaveICCProfile">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save current ICC profile in PNG file&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-left: 30px</string>
     </property>
     <property name="text">
      <string>Save ICC profile</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Compression level</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="cbIndexed">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If possible, try to save PNG file as indexed from a color palette&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-left: 30px</string>
     </property>
     <property name="text">
      <string>Save as indexed PNG, if possible</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSlider" name="hsCompressionLevel">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;PNG file format is a lossless format: compression size doesn't impact quality of saved image&lt;/p&gt;&lt;p&gt;Smallest value means low compression, bigger file size&lt;/p&gt;&lt;p&gt;Highest calue means high compression, smaller file size&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>6</number>
     </property>
     <property name="sliderPosition">
      <number>6</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="sbCompressionLevel">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;PNG file format is a lossless format: compression size doesn't impact quality of saved image&lt;/p&gt;&lt;p&gt;Smallest value means low compression, bigger file size&lt;/p&gt;&lt;p&gt;Highest calue means high compression, smaller file size&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
     <property name="value">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WColorButton</class>
   <extends>QToolButton</extends>
   <header>.pktk.widgets.wcolorbutton</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>hsCompressionLevel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sbCompressionLevel</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>568</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>617</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbCompressionLevel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hsCompressionLevel</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>617</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>568</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
